src/components/CommentSummaryButton/index.tsx

```diff
@@ -52,14 +46,7 @@ export const CommentSummaryButtonBase = ({
           </Popover.Content>
         </>
       }
-      onHide={() => {
-        dispatch(commentsActions.hideSummaryOverlay());
-      }}
-      onOpen={() => {
-        dispatch(commentsActions.toggleSummaryOverlay());
```

Comment:
I didn't port over this *toggle* action because technically it's not necessary. The button still works as expected because of how `Overlay` keeps track of its state internally and triggers the right callback.